content/models.py

```diff
@@ -403,6 +404,19 @@ def get_current(cls, user=None):
 
         return q.first()
 
+    def is_user_a_winner(self, querying_user_id):
+        """Checks to see whether or not a participant has been marked as a winner"""
+        if not self.is_active:
+            participant = Participant.objects.filter(user_id=querying_user_id)
```

Comment:
This is not correct. `filter()` will return a new queryset that contains multiple objects, not a single object. Use `get()` instead. I suggest catching `Participant.DoesNotExist` and returning `False` if the user has not participated.

```python
try:
    participant = Participant.objects.get(user_id=querying_user_id)
except Participant.DoesNotExist:
    return False
```

https://docs.djangoproject.com/en/1.10/ref/models/querysets/#filter
https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.get
https://docs.djangoproject.com/en/1.10/ref/models/instances/#django.db.models.Model.DoesNotExist